src/main/java/com/report/portal/business/page/objects/DashboardPage.java

```diff
@@ -0,0 +1,124 @@
+package com.report.portal.business.page.objects;
+
+import com.codeborne.selenide.Condition;
+import com.report.portal.business.constants.StepLabel;
+import com.report.portal.business.constants.WidgetType;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$;
+import static java.lang.String.format;
+
+public class DashboardPage extends BasePage {
+
+    private static final String widgetHeaderXpath = "//div[@class='widgetHeader__widget-name--FjJLi']//div[text()='%s']";
```

Comment:
According to Java Naming Convention (i.e. https://www.javatpoint.com/java-naming-conventions) change it to WIDGET_HEADER_XPATH

Reply:
changed. thanks!